### DISCUSSION TITLE
It appears MauiMaps initialization needs to change to work with the latest packages to run with Windows

### DISCUSSION THREAD

**Discussion by davefxy** [VladislavAntonyuk/MauiSamples#430] [Q&A] [answered]
I had to modify MauiProgram.cs. If we use .UseMauiMaps() with Windows the app will get an exception. So the following changes are needed:
`#if IOS || ANDROID
		builder.UseMauiMaps();
#endif

#if WINDOWS
		// Initialize the .NET MAUI Community Toolkit Maps by adding the below line of code
		builder.UseMauiCommunityToolkitMaps(Key);
#endif
`

**Comment by VladislavAntonyuk** [comment, marked as answer]
Fixed, starting from MCT.Maps v2, you need to call UseMauiMaps for all platforms

**Comment by davefxy** [comment]
I modified the previous code changes in order to work with Windows and Android. They are :
`
#if IOS || ANDROID
        .UseMauiMaps()
#endif
        .UseMauiCommunityToolkitMaps(BingMapsApiKey)
`
These are the nuget packages I am using:

		"CommunityToolkit.Maui" Version="8.0.1" 
		"CommunityToolkit.Maui.Maps" Version="2.0.2"
		"CommunityToolkit.Mvvm" Version="8.2.2"
		"Microsoft.Maui.Controls" Version="8.0.21"
		"Microsoft.Maui.Controls.Compatibility" Version="8.0.21"

Without these changes I made Windows version will immediately generate an error indicating map control is not supported in .Net Maui 8
